web_tree_image/README.rst

```diff
@@ -28,11 +28,14 @@ This module defines a new widget type for tree views columns.
 
 Set the attribute ``widget=image`` in a ``field`` tag in a tree view.
 You can also set ``height=<height>`` to set the height the image will have.
-Note that this just sets the CSS ``max-height`` attribute,
-if you want to make the server return a resized, maybe to save data by making it
-return a smaller one or to have uniform images, use the
+Note that this just sets the image ``height`` attribute,
+if you want to make the server return a resized image, maybe to reduce the size
+of the transferred file or to have uniform images, use the
 ``resize="<width>,<height>"`` attribute.
 
+You can set ``display=icon`` in order to get a clickable picture icon that will
```

Comment:
The use case for this were scanned documents, where a 16x16 thumbnail (or anything reasonably small to display in a tree view) wasn't particularly useful.
But if you think there are other use cases where this would be helpful, I can try to make it more flexible.